src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/Event.scala

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2016-2019 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.analytics.scalasdk
+
+// java
+import java.time.Instant
+import java.util.UUID
+
+// circe
+import io.circe.{Encoder, Json, JsonObject, ObjectEncoder}
+import io.circe.Json.JString
```

Comment:
Just noticed this one. `JString` is not public API AFAIK and it is not used anywhere. Make sure you're not importing unnecessary entities.